src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <ul class="calendar calendar--month-length--29 calendar--start-day--mon">
+        <li class="calendar__day calendar__day--1"></li>
```

Comment:
li should have only `calendar__day` class, you don't need `day--1, day--2` and so on